packages/runtime/src/api/constants.ts

```diff
@@ -1,26 +0,0 @@
-import type { DocumentNode } from 'graphql'
-
-import {
-  SwatchFragmentDoc,
-  FileFragmentDoc,
-  TypographyFragmentDoc,
-  PagePathnameSliceFragmentDoc,
-  GlobalElementFragmentDoc,
-  TableFragmentDoc,
-  SnippetFragmentDoc,
-  PageFragmentDoc,
-  SiteFragmentDoc,
-} from './generated/graphql'
-import type { APIResourceType } from './types'
-
-export const Fragments: Record<APIResourceType, DocumentNode> = {
-  Swatch: SwatchFragmentDoc,
-  File: FileFragmentDoc,
-  Typography: TypographyFragmentDoc,
-  PagePathnameSlice: PagePathnameSliceFragmentDoc,
-  GlobalElement: GlobalElementFragmentDoc,
-  Table: TableFragmentDoc,
-  Snippet: SnippetFragmentDoc,
-  Page: PageFragmentDoc,
-  Site: SiteFragmentDoc,
-}
```

Comment:
This file can't be deleted because the `Fragments` export is depended on by the builder to extract the data from the cache that it sends to the builder.